packages/z2s/src/sql.test.ts

```diff
@@ -72,9 +76,37 @@ describe('json arg packing', () => {
       ),
     ).toMatchInlineSnapshot(`
       {
-        "text": "SELECT * FROM "foo" WHERE "a" = $1->0 OR "b" = ($1->>1)::numeric OR "c" = $1->>2 OR "d" = ($1->>3)::boolean",
+        "text": "SELECT * FROM "foo" WHERE "a" = $1::json->0 OR "b" = ($1::json->>1)::numeric OR "c" = $1::json->>2 OR "d" = ($1::json->>3)::boolean",
```

Comment:
I'm concerned that casting to json multiple times may parse the text to json multiple times... 

What if instead of packing all the args into a single json array, we pass them each as a separate text arg?

Reply:
true, we don't need the json indirection.